### ISSUE TITLE
[SUGGESTION] Thrown weapons working with the Loyalty enchantment

### ISSUE THREAD

**Issue by DragonacyCX** [ThexXTURBOXx/balkons-weaponmod#91]
Just like the title says, I think it would be great if it to worked on knives and spears as well, even on the boomerang when it would normally get stuck in the ground. If it's possible, of course. Tridents for an instance have a nice feature of always returning to your offhand if they were here, but when it comes to balkon's weapons they often get lost in the player's inventory. Most of the enchantments work fine with the new weapons (fiery rounds with blunderbuss are fun) but if you plan on polishing the compatibility with enchants, that's the first idea that has come to my mind.

**Comment by Stormm232** [comment]
Are these so called knifes, supposed to be almost "**unlimited**" in throwing?
 Because my Musket broke and the Bayonet came off too, meaning, when I click right to throw I can throw 10-15 out of 1 knife.

https://imgur.com/a/2flc2Sl
I can pick all of them up, literally duplicating 1 knife into 10-15 at least.  

@ThexXTURBOXx

**Comment by ThexXTURBOXx** [comment]
@Stormm232 No, it's a bug, which has been reported several times already. Will be fixed in the next update

**Comment by ThexXTURBOXx** [comment]
I will implement this now. However, one small thing:

> Tridents for an instance have a nice feature of always returning to your offhand if they were here

I just tested this, but it does not work on my end. It's probably some mod, which does this

**Comment by ThexXTURBOXx** [comment]
Since this is a bit more complicated than I had anticipated, I will postpone this for now. I want to release 1.23.0 rather sooner than later and this would be a nice thing for 1.24.0, maybe :)

**Comment by ThexXTURBOXx** [comment]
I have implemented this successfully, but it is too hard to balance this. I tried nerving the enchantment a bit, but it still is way too OP.
I guess the trident enchantments really should stay with just the trident...
